Samples/src/OracleSynthesis/Program.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Quantum.Simulation.Core;
+using Microsoft.Quantum.Simulation.Simulators;
+
+namespace Microsoft.Quantum.Samples.OracleSynthesis {
+    class Program {
+        static void Main(string[] args) {
+            var sim = new QuantumSimulator();
+
+            for (var func = 0; func < (1 << 8); ++func) {
```

Comment:
It would be good to add a line 
```
var res = OracleSynthesis.Run(sim, func, 3).Result;
``` 
for the case of a non-clean target qubit.

Reply:
Added.